rfcs/text/0000-image-base-transcriptomics.md

```diff
@@ -0,0 +1,507 @@
+that images are not blank, contain spots, or contain cells, however
+these checks take increasing amounts of time.
+
+OME and SpaceTx can provide a docker container that validates image
+datasets with the highest level of stringency possible in a DCP-provided
+"maximum validation time".
+
+#### Output Data validation
+
+Each of the output formats produced by SpaceTx can be stored in Zarr and
+read (on-demand) by *starfish* and xarray. SpaceTx provides a docker
+container that loads and validates each output file using the xarray
+library. Validation can be tuned to occur within a DCP-provided target
+validation time.
+
+### Data Processing Workflows
+
+To process data in the DCP, the *starfish* team can deliver WDL-wrapped
+data processing workflows for any of the existing image-based
+transcriptomics pipelines. These workflows are based on the *starfish*
+python library, which in turn leverages standard open-source python
+projects. *starfish* provides a docker container that exposes both a
+python API and a CLI, and supports composable, modular, pipelines. These
+pipelines take as inputs images in SpaceTx Format and a pipeline recipe.
+Each assay emits output in the formats specified above. More detail on
+the state of image-based transcriptomics analyses can be found
+[<span class="underline">here</span>](https://docs.google.com/document/d/18Ve9C_cEn5lI2paivfgTAyjsM62qMyUF_s4mj82l1L0/edit#).
+
+### Data Processing Infrastructure
+
+All *starfish* pipelines share the same inputs and outputs. It should
+therefore be possible to deliver a single adapter WDL that satisfies any
+SpaceTx assay, and a family of subscriptions with minimal inter-class
+changes that specify the particular assay in question.
+
+### Delivery of Output Data to Users
+
+#### Matrices
+
+As mentioned above, image-based transcriptomics output data
+include a gene expression matrix and related metadata that are
+compatible with matrix service requirements and should be no
+more difficult to convert than the output of a sequencing assay
+
+However, other imaging experiments will likely require extending how
+features (genes) are named to support a broader set of features
+(eg. proteins and cellular components). Currently, the DCP standardizes
+around GENCODE gene IDs and HGNC gene symbols. Image based proteomics
+will require protein identifiers such as UNIPROT, and other use cases
+that measure sub-cellular components will likely require more general
+cellular component information (e.g. "endoplasmic reticulum").
+
+### **Acceptance Criteria**
+
+Except where listed ("responsible team"), the starfish and OME teams
+will do the majority of project management and execution for the
+following deliverables. They will coordinate and collaborate with DCP
+partners ("contact person") to validate design correctness and minimize
+code changes and their impact.
+
+| Component/Deliverable                                 	| Responsible Team/Service 	| No earlier than 	| Contact Person	|
```

Comment:
May I suggest that you eliminate the _No earlier than_ column and treat this table as required deliverables that will get scheduled (and perhaps get their own technical RFC) based on roadmap priorities? Per the RFC process - *NOTE*: _The existence of an approved software RFC does not indicate any particular priority or commitment to implement the RFC, nor is an approved RFC a green light to implement. Approved RFCs simply demonstrate community agreement on a technical decision. Product priorities are managed and implementations scheduled via the DCP PM planning process_ - and an explanation about "?" would be helpful.

Reply:
Done.